### PR TITLE
Improve code formatting and optimize imports

### PR DESCRIPTION
**Pull request by TobiGr**
I ran IntelliJ's "reformat code" command, but didn't pick all changes.

### CODE REVIEW

File: extractor/src/main/java/org/schabi/newpipe/extractor/MediaFormat.java [hunk 1]
```diff
@@ -116,25 +116,27 @@ public static MediaFormat getFromMimeType(String mimeType) {
 
     /**
      * Get the media format by it's id.
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/MediaFormat.java]: its

File: extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfo.java [hunk 2]
```diff
@@ -10,20 +8,22 @@
...
 import org.schabi.newpipe.extractor.utils.ExtractorHelper;
 
-public class CommentsInfo extends ListInfo<CommentsInfoItem>{
+import java.io.IOException;
+
+public class CommentsInfo extends ListInfo<CommentsInfoItem> {
+
+    private CommentsInfo(int serviceId, ListLinkHandler listUrlIdHandler, String name) {
+        super(serviceId, listUrlIdHandler, name);
+        // TODO Auto-generated constructor stub
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfo.java]: We should remove this

File: extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/ListLinkHandlerFactory.java [hunk 3]
```diff
@@ -53,18 +59,19 @@ public ListLinkHandler fromQuery(String id,
...
                                      List<String> contentFilters,
                                      String sortFilter, String baseUrl) throws ParsingException {
         final String url = getUrl(id, contentFilters, sortFilter, baseUrl);
         return new ListLinkHandler(url, url, id, contentFilters, sortFilter);
     }
 
-    
+
     /**
      * For makeing ListLinkHandlerFactory compatible with LinkHandlerFactory we need to override this,
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/ListLinkHandlerFactory.java]: making

File: extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/SearchQueryHandlerFactory.java [hunk 4]
```diff
@@ -35,9 +40,12 @@ public SearchQueryHandler fromQuery(String querry) throws ParsingException {
 
     /**
      * It's not mandatorry for NewPipe to handle the Url
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/SearchQueryHandlerFactory.java]: mandatory

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubePlaylistExtractor.java [hunk 5]
```diff
@@ -74,7 +74,7 @@ public String getNextPageUrl() throws IOException, ExtractionException {
     @Override
     public void onFetchPage(Downloader downloader) throws IOException, ExtractionException {
         // TODO Auto-generated method stub
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubePlaylistExtractor.java]: remove this

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java [hunk 6]
```diff
@@ -68,7 +68,7 @@ public String getThumbnailUrl() throws ParsingException {
     @Override
     public String getBannerUrl() {
         return "";      // Banner can't be handled by frontend right now.
-                        // Whoever is willing to implement this should also implement this in the fornt end
+        // Whoever is willing to implement this should also implement this in the fornt end
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java]: front

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 7]
```diff
@@ -125,6 +130,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
     /**
      * The Amount of likes a video/audio stream got.
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: amount

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 8]
```diff
@@ -133,6 +139,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
     /**
      * The Amount of dislikes a video/audio stream got.
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: amount

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 9]
```diff
@@ -259,6 +279,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
      * should return the url of the next stream. NewPipe will automatically play
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: Should

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 10]
```diff
@@ -271,6 +292,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
      * be disabled by the user later in the frontend.
      * This list MUST NOT contain the next available video as this should be return through getNextStream()
      * If  is is not available simply return null
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: two spaces instead of one

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 11]
```diff
@@ -302,6 +325,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
     /**
      * Override this function if the format of time stamp in the url is not the same format as that form youtube.
      * Honestly I don't even know the time stamp fromat of youtube.
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: format

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 12]
```diff
@@ -302,6 +325,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
...
+     *
      * @param regexPattern
      * @return the sime stamp/seek for the video in seconds
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: time, not sime

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 13]
```diff
@@ -302,6 +325,7 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
...
      * @throws ParsingException
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: below:
// -2 because well the testing system will then know its the regex that failed :/
will consequently know that the regex failed

**Comment by B0pol** [comment]
BTW, in [copyright file](https://github.com/TeamNewPipe/NewPipeExtractor/blob/dev/copyright), the email is chris.schabesberger@mailbox.com, but in every other file, it's chris.schabesberger@mailbox.org

**Comment by Stypox** [comment]
@B0pol you can review all files at once: just press "Start Review" on the first comment and add comments until you are done, then click "Submit Review"

**Comment by Stypox** [comment]
Also, for single-line suggestions you can click on the central file symbol in the comment toolbar to make a commit-able suggestion

File: extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfo.java (hunk 2 above)
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfo.java]:
```suggestion
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java (hunk 10 above)
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]:
```suggestion
     * If it is not available simply return null
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java (hunk 11 above)
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]:
```suggestion
     * Honestly I don't even know the time stamp format of YouTube.
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java (hunk 12 above)
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]:
```suggestion
     * @return the time stamp/seek for the video in seconds
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/MediaFormat.java (hunk 1 above)
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/MediaFormat.java]:
```suggestion
     * Get the media format by its id.
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java (hunk 13 above)
Comment (TobiGr) [reply extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: I don't understand what you mean. Can you explain in more detail please?

Comment (B0pol) [reply extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: Below this line there is a comment
``// -2 because well the testing system will then know its the regex that failed``
I suggest to reword it to
``// -2 because the testing system will consequently know that the regex failed``

**Comment by TobiGr** [comment]
@B0pol Thanks!

I'll ask @theScrabi about the email address, but afaik this should be `.org`.

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java (hunk 13 above)
Comment (TobiGr) [reply extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]: Done

**Comment by TobiGr** [comment]
@B0pol @Stypox Is this good to merge?

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/PeertubeInstance.java [hunk 14]
```diff
@@ -10,45 +11,43 @@
...
+
         try {
             response = downloader.get(url + "/api/v1/config");
         } catch (ReCaptchaException | IOException e) {
             throw new Exception("unable to configure instance " + url, e);
         }
-        
-        if(null == response || StringUtil.isBlank(response.responseBody())) {
+
+        if (null == response || StringUtil.isBlank(response.responseBody())) {
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/PeertubeInstance.java]:
```suggestion
        if (response == null || StringUtil.isBlank(response.responseBody())) {
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeChannelExtractor.java [hunk 15]
```diff
@@ -172,14 +171,14 @@ private void setInitialData(String responseBody) throws ExtractionException {
         } catch (JsonParserException e) {
             throw new ExtractionException("Unable to extract peertube channel data", e);
         }
-        if(null == json) throw new ExtractionException("Unable to extract peertube channel data");
+        if (null == json) throw new ExtractionException("Unable to extract PeerTube channel data");
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeChannelExtractor.java]:
```suggestion
        if (json == null) throw new ExtractionException("Unable to extract PeerTube channel data");
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java [hunk 16]
```diff
@@ -107,32 +108,32 @@ private String getNextPageUrl(String continuation) throws ParsingException {
...
+            if (c instanceof JsonObject) {
                 CommentsInfoItemExtractor extractor = new YoutubeCommentsInfoItemExtractor((JsonObject) c, getUrl(), getTimeAgoParser());
                 collector.commit(extractor);
             }
         }
     }
 
     private void fetchTitle(JsonArray contents) {
-        if(null == title) {
+        if (null == title) {
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java]:
```suggestion
        if (title == null) {
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java [hunk 17]
```diff
@@ -68,7 +68,7 @@ public String getThumbnailUrl() throws ParsingException {
     @Override
     public String getBannerUrl() {
         return "";      // Banner can't be handled by frontend right now.
-                        // Whoever is willing to implement this should also implement this in the fornt end
+        // Whoever is willing to implement this should also implement this in the front end.
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java]:
```suggestion
        // Whoever is willing to implement this should also implement it in the frontend.
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 18]
```diff
@@ -313,7 +337,7 @@ protected long getTimestampSeconds(String regexPattern) throws ParsingException
         } catch (Parser.RegexException e) {
             // catch this instantly since an url does not necessarily have to have a time stamp
 
-            // -2 because well the testing system will then know its the regex that failed :/
+            // -2 because the testing system will consequently know that the regex failed
             // not good i know
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]:
```suggestion
            // not good, I know
```

File: extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java [hunk 19]
```diff
@@ -313,7 +337,7 @@ protected long getTimestampSeconds(String regexPattern) throws ParsingException
         } catch (Parser.RegexException e) {
             // catch this instantly since an url does not necessarily have to have a time stamp
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java]:
```suggestion
            // catch this instantly since a url does not necessarily have a timestamp
```

**Comment by B0pol** [comment]
You should close #225
